src/frequenz/sdk/timeseries/_formula_engine/_formula_generators/_formula_generator.py

```diff
@@ -31,6 +31,7 @@ class ComponentNotFound(FormulaGenerationError):
 
 
 NON_EXISTING_COMPONENT_ID = sys.maxsize
+"""A component ID that does not exist in the component graph."""
```

Comment:
Maybe add some extra info? when this should be used?

Reply:
Added extra info. @llucax please review the recently added documentation